benchmark/components/simd_fnt.cpp

```diff
@@ -0,0 +1,370 @@
+/*
+ * Copyright 2017-2019 Scality
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice,
+ *    this list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * 3. Neither the name of the copyright holder nor the names of its contributors
+ *    may be used to endorse or promote products derived from this software
+ *    without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+ * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ */
+#include <functional>
+#include <vector>
+
+#include <benchmark/benchmark.h>
+
+#include "arith.h"
+#include "core.h"
+#include "gf_prime.h"
+#include "misc.h"
+#include "vec_buffers.h"
+#include "vec_vector.h"
+
+namespace arith = quadiron::arith;
+namespace vec = quadiron::vec;
+namespace gf = quadiron::gf;
+
+#ifdef QUADIRON_USE_SIMD
+
+#include "simd.h"
+#include "simd/simd.h"
+#include "simd_fnt.h"
+
+namespace simd = quadiron::simd;
+
+// Range of data size for tests
+const size_t MIN_VEC_LEN = 1;
```

Comment:
For those `const` you could use `constexpr` instead (since they re compile-time constant).

Reply:
Done. Thanks